src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -178,6 +178,7 @@ describe('sidebar.components.sidebar-content', function() {
   describe('clearSelection', () => {
     it('sets selectedTab to Annotations tab if selectedTab is null', () => {
       store.selectTab(uiConstants.TAB_ORPHANS);
+      store.setDirectLinkedGroupFetchFailed();
```

Comment:
This change doesn't appear to have anything to do with the condition that the test is checking for.